11-query/05-updating-data.md

```diff
@@ -112,46 +115,46 @@ GraqlInsert insert_query = Graql.match(
 [tab:end]
 </div>
 
-However, to modify or delete role players, the current limitations require us to [delete the instances of the relation](../11-query/04-delete-query.md#delete-instances-of-a-relation-type) with the current role players and [insert the new instance of the relation](../11-query/03-insert-query.md#insert-instances-of-a-relation-type) with the new roleplayers.
+To replace a role player, we combine extending the relation with removing a role player:
```

Comment:
hmmm, doesn't read so well. feels like too many verbs and infinitives in a short sentence. could you paraphrase this to make it easier to read and understand?

Reply:
added a couple words

Reply:
that's better and I understand what you mean by it, after reading the query. 

what I think could be even better is if we breakdown the [Update a relation by changing its role players](https://github.com/graknlabs/docs/blob/3b7d030f6e761ad0691600f7bb1d76069c3ea03d/11-query/05-updating-data.md#update-a-relation-by-changing-its-role-players) use-case to 3 use-cases that are more specific -- to add, remove and replace a role player. these are probably things that people search for, and it'd be very helpful if they're headings. also if they are documented in that order, I think it'll be easier to understand the replace use-case.